### DISCUSSION TITLE
java.lang.NoSuchMethodError: org.opencv.imgcodecs.Imgcodecs.dicomJpgMatRead

### DISCUSSION THREAD

**Discussion by Ronaldoshuang** [dcm4che/dcm4che#1090] [Q&A]
**Describe the bug**
this method run  throws java.lang.NoSuchMethodError: org.opencv.imgcodecs.Imgcodecs.dicomJpgMatRead，
but idea still add jar opencv-456.jar, i do not konw what happen？how can i do ？
`public static void transcodeWithTranscoder(File src, final File dest) throws IOException {
        try (Transcoder transcoder = new Transcoder(src)) {
            transcoder.setDestinationTransferSyntax(UID.ExplicitVRLittleEndian);
            transcoder.transcode(new Transcoder.Handler(){
                @Override
                public OutputStream newOutputStream(Transcoder transcoder, Attributes dataset) throws IOException {
                    return new FileOutputStream(dest);
                }
            });
        } catch (Exception e) {
            Files.deleteIfExists(dest.toPath());
            throw e;
        }
    }`
![image](https://user-images.githubusercontent.com/23060328/140277427-1e397359-b10c-41f2-beb6-c7705cf24cd4.png)

**Comment by nroduit** [comment]
See this [post](https://groups.google.com/g/dcm4che/c/IFaXh9l2KqE/m/OBmpPWayAwAJ)
